backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -145,6 +144,54 @@ class SchedulingService implements ISchedulingService {
     return schedulingDtos;
   }
 
+  async getSchedulingsByVolunteersNeeded(
+    isVolunteerSlotFilled?: boolean,
+  ): Promise<SchedulingDTO[]> {
+    let schedulingDtos: Array<SchedulingDTO> = [];
+    try {
+      const schedulings: Array<Scheduling> = await Scheduling.findAll({
+        where:
```

Comment:
I wonder if there's is an easier way to deal with this? We tried making a nested ternary but that caused a linting error :( can we turn that linting rule off LOL

Reply:
why not just lift the query out of the nested ternary?

```typescript
 const volunteerIdQuery = isVolunteerSlotFilled ? { [Op.ne]: null } : null;
 const schedulings: Array<Scheduling> = await Scheduling.findAll({
        where:
          isVolunteerSlotFilled === undefined
            ? {
                volunteer_needed: true,
              }
            : {
                volunteer_needed: true,
                volunteer_id: volunteerIdQuery,
              },
        order: [["start_time", "ASC"]],
      });
```

Reply:
just did that :)